docs/serverless-v2/01-01-serverless-v2.md

```diff
@@ -2,7 +2,7 @@
 title: Overview
 ---
 
->**CAUTION:** This implementation will soon replace Serverless that is based on [Kubeless](https://github.com/kubeless/kubeless). Serverless v2 relies on [Knative Serving](https://knative.dev/docs/serving/) for deploying and managing functions, and [Tekton Pipelines](https://github.com/tektoncd/pipeline) for creating and running cluster-wide processes. Consider this implementation as experimental as it is still under development.
+>**CAUTION:** This implementation will soon replace Serverless that is based on [Kubeless](https://github.com/kubeless/kubeless). Serverless v2 relies on [Knative Serving](https://knative.dev/docs/serving/) for deploying and managing functions, and [Kubernetes Jobs](https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/) for creating Docker images. Consider this implementation as experimental as it is still under development.
 
 "Serverless" refers to an architecture in which the infrastructure of your applications is managed by cloud providers. Contrary to its name, serverless applications do require a server but they don't require you to run and manage it on your own. Instead, you subscribe to a given cloud provider, such as AWS, Azure or GCP, and pay a subscription fee only for the resources you actually use. Since the resource allocation can be dynamic and depends on your current needs, the serverless model is particularly cost-effective when you want to implement a certain logic that is triggered on demand. Simply, you get your things done and don't pay for the infrastructure that sits idle.
```

Comment:
I know that this is not the scope of the review :D but I have a question about this part:
`Contrary to its name, serverless applications do require a server but they don't require you to run and manage it on your own.` Shouldn't it be `contrary to their name`? as you are speaking about the `applications` later on.
Or `contrary to its name, a serverless application does...`

Reply:
True, I'll change that :)